### ISSUE TITLE
Status of the project

### ISSUE THREAD

**Issue by julweber** [cloudfoundry-community/logsearch-boshrelease#189]
Hi @axelaris,

i just wanted to ask whether there are any plans on continuing this release?

What i expect as an answer is a "no", when i look at the last changes made to the repo.
Is something out there in the open source community that could be used as a replacement.

Maybe you could point me to something, as we are struggling with switching to something else from this release in our systems.

Kind regards and thanks in advance
Julian Weber

**Comment by axelaris** [comment]
Hi Julian.
This project was supported by the company where I worked before. Unfortunately, it was suffered because of Covid, and now does not have resources to support open source projects :-(

Sorry to say that, but currently this project can be treated as orphaned. If anyone want to continue develop it - I would say thanks and give necessary permissions.

**Comment by julweber** [comment]
Hi @axelaris,

thanks for the fast feedback.

This is what we expected. 

I wish you fun and good luck with your new projects.
Thanks for maintaining this project in the past.

Kind regards,
Julian

**Comment by amhuber** [comment]
@bengerman13, is taking over this repo something that your organization would be able to take on?  I've been eyeing your fork at https://github.com/cloud-gov/logsearch-boshrelease/ for some time trying to decide if we need to switch over to it for our deployments, but I'm not sure if that is intended for general use or if it's been customized for your use case.  Given your current pending pull request as well it seems like you're the best candidate to keep this project current.

I've got very limited development resources but I would be happy to offer testing resources to make sure your new pull request doesn't cause any issues for a generic deployment.

**Comment by bengerman13** [comment]
@amhuber I'll discuss it with our team. Currently, most of our efforts are based on security and compliance findings, rather than on features/improvements. I don't believe we have any super-specific modifications - the only thing I believe we've done other than version bumps is permissions fixes as ES and Kibana change APIs. 

(Aside: if you want to follow our fork for the time being, know that we currently track `.final_builds` and `releases` directories outside of git, but you'd probably have to do the same since our blobstore is private)

**Comment by amhuber** [comment]
TBH that's really all we need (security fixes vs. new features).  The existing release desperately needs your pull request to get the latest JDK and ELK security fixes.

**Comment by bengerman13** [comment]
We're still looking into this, but right now our main concern is about licensing. Because of the changes in Elastic's licensing strategy, we think we need to either:
- move to an option with more permissive licensing (e.g. opensearch)
- build elasticsearch components from source, and _change the license to [SSPL](https://www.mongodb.com/licensing/server-side-public-license)_ wherever we do so

I think moving to to a different backend (such as opensearch) is a major breaking change, and the migration would be nontrivial for most or all users, so I'm somewhat inclined to do that in a new repo, probably a hard fork of our current logsearch-boshrelease.

**Comment by amhuber** [comment]
Yeah, I've been wondering when (not if) the licensing change was going to become a problem.  Thanks in advance for considering this, and I suspect many in the CF community would be grateful (and hopefully contribute to) a fork based on OpenSearch with the same functionality.

**Comment by bengerman13** [comment]
We're getting closer to owning a logging stack on our end. We've sketched out a plan [here](https://github.com/cloud-gov/product/pull/1744) and welcome input from the community.

Because we're being driven by compliance and security considerations, we'll be running this in our github org and with internal governance at least for the time being, but our code is all licensed CC0, so it's readily forkable if anyone needs something we can't incorporate/accommodate. 

With this breadcrumb dropped, we're going to try to keep any conversation about our projects on our own repos to keep potential confusion and unwanted noise down.

**Comment by markdboyd** [comment]
FYI, we have started creating a BOSH release for Opensearch + Opensearch Dashboards:

https://github.com/cloud-gov/opensearch-boshrelease

The intent is that this BOSH release will replicate the functionality of these Logsearch repos:

- https://github.com/cloudfoundry-community/logsearch-boshrelease
- https://github.com/cloudfoundry-community/logsearch-for-cloudfoundry

**Comment by amhuber** [comment]
Awesome!  I'm sure we're not the only ones in the community who will benefit greatly from that.
